File: backend/Entities/Models/Documento1.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Documento1
{
    public Guid Id { get; set; }

    public Guid DocumentoId { get; set; }

    public Guid TenantId { get; set; }

    public DateTime DataEntrada { get; set; }

    public bool? Ativo { get; set; }

    public virtual Documento Documento { get; set; } = null!;

    public virtual DocumentoFilialGrupo? DocumentoFilialGrupo { get; set; }

    public virtual RomaneioDocumento? RomaneioDocumento { get; set; }

    public virtual Tenant Tenant { get; set; } = null!;
}

File: frontend-with-menus/src/app/Business/Models/documento1.ts
import { Documento } from "./dcumento";
import { DocumentoFilialGrupo } from "./documentoFilialGrupo";
import { RomaneioDocumento } from "./romaneioDocumento";
import { Tenant } from "./tenant";

export interface Documento1  {
    id: string;
    documentoId: string;
    tenantId: string;
    dataEntrada: string;
    ativo: boolean | null;
    documento: Documento;
    documentoFilialGrupo: DocumentoFilialGrupo | null;
    romaneioDocumento: RomaneioDocumento | null;
    tenant: Tenant;
}